Sources/Rendering/Core/RenderWindowInteractor/index.js

```diff
@@ -940,6 +940,19 @@ function vtkRenderWindowInteractor(publicAPI, model) {
     }
     superDelete();
   };
+
+  // Use the Page Visibility API to detect when we switch away from or back to
+  // this tab, and reset the lastFrameStart. When tabs are not active, browsers
+  // will stop calling requestAnimationFrame callbacks.
+  if (typeof document.hidden !== 'undefined') {
+    document.addEventListener(
```

Comment:
Probably a good idea to remove the event listener in `delete()`.

Reply:
:+1: good idea